assets/js/modules/reader-revenue-manager/components/settings/SettingsEdit.test.js

```diff
@@ -393,4 +403,82 @@ describe( 'SettingsEdit', () => {
 			).not.toBeInTheDocument();
 		} );
 	} );
+
+	describe( 'GA event tracking', () => {
+		afterEach( () => {
+			mockTrackEvent.mockClear();
+		} );
```

Comment:
We typically prefer not to use `describe()` to group test cases unnecessarily, and this `afterEach()` would sit well at the top level. Please can you move it up and remove the `describe()`?

Reply:
Updated, thanks Tom!